src/services/index.ts

```diff
@@ -0,0 +1,15 @@
+import GamesService from './games-service';
+import StudiosService from './studios-service';
+import UsersService from './users-service';
+
+export interface Services {
```

Comment:
hm what is advantage of such interface?

To ask it in simple way: why not simple inject exact service we need?

Reply:
yes, its not needed, I just experimented, tried to first define interface